src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/prisonersearch/PrisonerSearchClient.kt

```diff
@@ -32,4 +32,6 @@ class PrisonerValidator(val prisonerSearchClient: PrisonerSearchClient) {
 data class Prisoner(
   val prisonerNumber: String,
```

Comment:
Isn't there a model for prisoner? OffenderDetail - something like that?

Reply:
No not at the moment, this is just the model from the search API.  Can add one though.

Reply:
Done